content/tracing/setup/dotnet.md

```diff
@@ -52,25 +52,92 @@ net start w3svc
 
 .NET Core doesn't support the Windows Global Assembly Cache (GAC) used by the .NET Framework. Instead, you can deploy Datadog's managed assemblies together with your application by adding the `Datadog.Trace.ClrProfiler.Managed` NuGet package. It is important that you match the versions of the NuGet package and the MSI installer.
 
-For example, if you are using the MSI installer for version `0.4.1-beta`, use:
+For example, if you are using the MSI installer for version `0.5.0-beta`, use:
 
 ```
-dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.4.1-beta
+dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.5.0-beta
 ```
 
 ### Linux
 
-Automatic instrumention for .NET Core on Linux is coming soon. To instrument your .NET Core applications on Linux today, use [manual instrumentation][8].
+Automatic instrumention is available on Linux for .NET Core 2.0+. To setup automatic instrumentation, 4 steps are necessary:
+
+1. [Install the Datadog Agent][1]
+1. Add the `Datadog.Trace.ClrProfiler.Managed` nuget package to your project:
+
+   ```bash
+   dotnet add package Datadog.Trace.ClrProfiler.Managed --version 0.5.0-beta
+   ```
+
+1. Install the `Datadog.Trace.ClrProfiler.Native` library, available from the `dd-trace-csharp` [releases page][10].
+
+   For Debian or Ubuntu, download and install the Debian package:
+
+   ```bash
+   curl -LO https://github.com/DataDog/dd-trace-csharp/releases/download/v0.5.0-beta/datadog-dotnet-apm_0.5.0_amd64.deb
+   sudo apt install ./datadog-dotnet-apm_0.5.0_amd64.deb
+   ```
+
+   For CentOS or Fedora, download and install the RPM package
+
+   ```bash
+   curl -LO https://github.com/DataDog/dd-trace-csharp/releases/download/v0.5.0-beta/datadog-dotnet-apm-0.5.0-1.x86_64.rpm
+   sudo rpm -Uvh datadog-dotnet-apm-0.5.0-1.x86_64.rpm
+   ```
+
+   A Tar archive is available for other distributions:
+
+   ```bash
+   sudo mkdir -p /opt/datadog
+   curl -L https://github.com/DataDog/dd-trace-csharp/releases/download/v0.5.0-beta/datadog-dotnet-apm-0.5.0.tar.gz \
+   | sudo tar xzf - -C /opt/datadog
+   ```
+
+   For Alpine Linux you will also need to install `libc6-compat`
+
+   ```bash
+   apk add libc6-compat
+   ```
+
+1. Add environment variables to your service to enable the .NET tracer.
+
+   For Systemd:
+
+   ```ini
+    [Unit]
+    Description=example
+
+    [Service]
+    ExecStart=/usr/bin/dotnet /app/example.dll
+    Restart=always
+    # Datadog .NET Tracer Environment Variables
+    Environment=CORECLR_ENABLE_PROFILING=1
+    Environment=CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+    Environment=CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
+    Environment=DD_INTEGRATIONS=/opt/datadog/integrations.json
+
+    [Install]
+    WantedBy=multi-user.target
+   ```
+
+   For Docker:
+
+   ```docker
+   ENV CORECLR_ENABLE_PROFILING=1
```

Comment:
This is drawing weird red lines before and after some lines (see [preview](https://docs-staging.datadoghq.com/caleb/dotnet-linux/tracing/setup/dotnet/)). Should we drop the `docker` formatting?